src/Spryker/Client/ProductLabelStorage/ProductLabelStorageClient.php

```diff
@@ -51,20 +51,28 @@ public function findLabels(array $idProductLabels, $localeName)
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      *
      * @api
      *
      * @param string $labelName
      * @param string $localeName
+     * @param string|null $storeName
      *
      * @return \Generated\Shared\Transfer\ProductLabelDictionaryItemTransfer|null
      */
-    public function findLabelByName($labelName, $localeName)
+    public function findLabelByName($labelName, $localeName, ?string $storeName = null)
     {
+        if (!$storeName) {
```

Comment:
Architect's decision to move the check with store name on client layer
https://github.com/spryker/spryker/pull/6714#discussion_r382542536